src/types.ts

```diff
@@ -122,3 +122,28 @@ export interface Quiz {
 export interface RawQuizzes {
   [key: string]: RawQuiz
 }
+
+type QuizLevel = "beginner" | "intermediate"
```

Comment:
Any reason to not just add `advanced` in here now?

```suggestion
type QuizLevel = "beginner" | "intermediate" | "advanced"
```

Reply:
@wackerow because we currently don't have any advanced quiz, YAGNI :P